indexing-service/src/main/java/org/apache/druid/indexing/overlord/TaskQueue.java

```diff
@@ -947,6 +947,25 @@ public CoordinatorRunStats getQueueStats()
     return stats;
   }
 
+  public Optional<Task> getTask(String id)
```

Comment:
I think it would be better for this method to return only active tasks, i.e. it shouldn't have to query the underlying storage on behalf of the caller. As such, this method should be renamed to `getActiveTask`

Reply:
Done